FirefoxAction.cs

```diff
@@ -101,7 +125,7 @@ internal static RegistryKey OpenOrCreateSubKey(this RegistryKey registryKey, str
 
         internal static string GetStringValue(this RegistryKey registryKey, string name, string defaultValue = null)
         {
-            if (!registryKey.ContainsName(name))
+            if (Array.IndexOf(registryKey.GetValueNames(), name) < 0)
```

Comment:
Would LINQ be more idiomatic here (or maybe not), also, let's use braces throughout:

```suggestion
            if (registryKey.GetValueNames().Any(name.Equals)) {
```